src/components/pullRequests/section/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { FunctionComponent } from 'react';
+import Card from '../card';
+import classNames from './section.module.scss';
+
+type Props = {
+  heading: string,
+  content: []
+}
+
+const Section: FunctionComponent<Props> = ({ heading, content }) => {
+
+  const cards = content.map((pullRequest, i) => {
+    return <Card pullRequest={pullRequest} key={i} />
+  })
```

Comment:
Cards will re-render and reinitialized every time `Section` updates
Can we make it a separate function and pull it out of the `Section` function definition?